README.md

```diff
@@ -6,13 +6,96 @@
+
+The Cloud Debugger is a feature of the Google Cloud Platform that lets you inspect the state of your application at any code location without stopping your application. For more information about the Cloud Debugger, see https://cloud.google.com/tools/cloud-debugger/. Here's an introductory video:
+
+[![Cloud Debugger Intro](http://img.youtube.com/vi/tyHcK_kAOpw/0.jpg)](https://www.youtube.com/watch?v=tyHcK_kAOpw)
+
+## Prerequisites
+* You are using Node.js 0.12+ for your application. (Node.js v5+ is recommended).
+* The source of your application is uploaded to your cloud project's [source repository](https://cloud.google.com/tools/cloud-repositories/docs/). The Debugger UI needs the source to be available in order to set breakpoints.
+* (Optional) You should already be familiar with deploying your Node.js applications on Google Cloud Platform. See [https://cloud.google.com/nodejs/](https://cloud.google.com/nodejs/) to get started.
+
+## Quick Start (Node.js v4.x+)
+```shell
+# Install module
+npm install --save @google/cloud-debug
+
+# Change the start command of your application in package.json
+   ...
+   "scripts" {
+     "start": "node --require @google/cloud-debug your-app.js"
+   ...
+
+# Deploy and debug your application
+```
+
+## Detailed instructions
+
+1. Install the debug agent module.
+
+   ```shell
+   npm install --save @google/cloud-debug
+   ```
+1. Ensure that the module gets loaded during your application's startup. The best way to do this is via the `--require` command line option available in Node.js v4. You should add this option to the startup script section in your `package.json`.
+
+   ```
+   "scripts": {
+      "startup": "node --require @google/cloud-debug server.js",
+   ```
+   If you are using Node.js v0.12.x, you will not be able to use the `--require` command line option. Instead you will have to manually load the debug agent module in the startup script of your application. For example, you may want to add the following
+   to your `server.js`:
+   ```javascript
+   require('@google/cloud-debug');
+   ```
+
+At this point you can deploy your application to AppEngine or to Google Compute Engine and the debug agent will activate with no further action. If you want to run your application locally, you can still use the cloud debug agent, with a few additional steps:
+
+1. The agent needs to know the numeric project id of your Google cloud project. You can obtain this information from the project's settings page in the Google Developers Console (click on the gear icon in the corner.)
+
+  ```shell
+  export GCLOUD_PROJECT_NUM=<your numeric project id>
+  ```
+1. You need to provide service account credentials in order for the debug agent to be able to authenticate to the debug API. To create a service account:
+  1. Visit the [Google Developers Console][dev-console].
```

Comment:
Maybe discussion of gcloud auth login here since they likely already use the gcloud tool.